src/main/java/com/hermesworld/ais/galapagos/topics/service/TopicService.java

```diff
@@ -116,7 +116,7 @@
             int limit);
 
     @CheckReturnValue
-    CompletableFuture<Void> addTopicProducers(String environmentId, String topicName, List<String> producers);
+    CompletableFuture<Void> addTopicProducer(String environmentId, String topicName, String producerId);
 
     @CheckReturnValue
     CompletableFuture<Void> removeProducerFromTopic(String envId, String topicName, String appId);
```

Comment:
Asymmetrical. Either `addTopicProducer` and `removeTopicProducer`, or (I wouldn't like that) `addProducerToTopic` and `removeProducerFromTopic`.